src/Monolog/Formatter/LogmaticFormatter.php

```diff
@@ -19,6 +23,11 @@
 class LogmaticFormatter extends JsonFormatter
 {
     /**
+     * @param array $marker Used by Logmatic.io to identify the technology
+     */
+    protected $marker = ["sourcecode", "php"];
```

Comment:
and if it cannot be changed du to Logmatic requirements, it should not be a property at all